package.json

```diff
@@ -53,7 +54,8 @@
     "seed": "sh db/seed.sh",
     "format": "prettier --write \"**/*.{js,jsx,json,md,css,scss}\"",
     "test": "jest --silent",
-    "test::verbose": "jest"
+    "test::verbose": "jest",
+    "test::e2e": "node_modules\\.bin\\cypress open"
```

Comment:
For CI purpose, running `cypress run` will be sufficient.

We could have two scripts:
```json
{
  "test::e2e": "cypress run",
  "cypress::open": "cypress open"
}
```

The first one is more general and will be run on every PRs. The second will be used if necessary when developing locally.

Reply:
Yep!! actually that makes sense.

Reply:
But, just a thought 💭. @reobin 
If running test locally while developing, if the developer runs `cypress::open` locally and does not have cypress installed in the  system, but installed in particular that environment(/node_modules).

Then, it will give an error saying `cypress command is not recognized` .
Then, developer has to run like `node_modules\\.bin\\cypress open` running. Right?
Please, correct me if I am wrong because, this is first time I am using cypress

Reply:
Since the script is run by `npm`, it will use the `cypress` that is installed in the `node_modules`, so a user should not have any problem running `"cypress::open": "cypress open"` even if it is not installed on the machine directly.

Reply:
Ahh, right! I missed that. Thank you😃